src/qt_gui/translations/pl_PL.ts

```diff
@@ -898,5 +898,10 @@
             <source>Failed to parse JSON:</source>
             <translation>Nie udało się przeanlizować JSON:</translation>
         </message>
+		<message>
+			<location filename="../cheats_patches.cpp" line="1163"/>
+			<source>Can't apply cheats before the game is started</source>
+			<translation>Nie można zastosować cheatów przed rozpoczęciem gry.</translation>
```

Comment:
```suggestion
			<translation>Nie można zastosować kodów przed uruchomieniem gry.</translation>
```

Keep translation consistent to other translated strings